prow/plugins/config.go

```diff
@@ -51,6 +51,9 @@ type Configuration struct {
 	// Owners contains configuration related to handling OWNERS files.
 	Owners Owners `json:"owners,omitempty"`
 
+	// ProwURL is the URL to use in help messages when referring to this deployment
+	ProwURL string `json:"prow_url"`
```

Comment:
Do we need to have two separate settings for this? (one in config.yaml, and another in plugins.yaml?)

Reply:
Not everyone loads both :(

It's awful, suggestions appreciated here (@cjwagner ?)

Reply:
That belongs in Prow config only (not plugin config) IMO. Core Prow config is also exposed to plugins and the Prow URL makes more sense in core config anyways.